src/firefly/java/edu/caltech/ipac/table/io/FITSExtractToTable.java

```diff
@@ -0,0 +1,265 @@
+package edu.caltech.ipac.table.io;
```

Comment:
This whole file was just refactored out of FITSTableReader, right?  So it's not a big a mod as it looks...